### DISCUSSION TITLE
How can I edit a POI linked to a favorit?

### DISCUSSION THREAD

**Discussion by bege10** [osmandapp/OsmAnd#18037] [Q&A] [answered]
When I create a favorit on a POI they are linked and I can see the details of the POI with the favorit. That's perfect but then the feature to edit the POI is gone. How can I edit the POI linked to a favorit in OsmAnd?

**Comment by pebogufi** [comment]
> When I create a favorit on a POI they are linked and I can see the details of the POI with the favorit. That's perfect but then the feature to edit the POI is gone.. ..

No, it is not.
Short tap on the object show both, poi and favorite. Select which one you want to edit.

Which version osmand and system do you use ?
Maybe you should update ?

**Comment by bege10** [comment]
> Short tap on the object show both, poi and favorite. Select which one you want to edit.

Not here. (Fairphone 3, /e/OS (based on LineageOS for microG, Android 12), OsmAnd 4.5.8 F-Droid). There is only the favorite and the action to edit the POI changes to create POI.


<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/1184237/b78055bc-0f9d-439c-93af-46ac69453a2a"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by DmitryAlexei** [reply]
- Close Favorite context menu and then 
- short tap on where POI/favorite should be located
- POI and favorite will available to select (see picture)
also note, that POI can be deleted, so it will not be present on map any more

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/487b4536-13e9-4c41-9ee0-3cbb36d97667"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by bege10** [reply]
Here his works if the favorite is close to the POI. It does not work if the favorite is placed right on the POI so that the POI details are displayed with the favorite as seen here https://github.com/osmandapp/OsmAnd/discussions/18037#discussioncomment-6870874.

**Reply by DmitryAlexei** [reply, marked as answer]
@bege10 As you can see on screenshot, *Etno Cafe Jednorozec* is a POI and a Favorite. It isn't close each to other, it is the same place.
As it was assumed, your POI was deleted, so it is no longer present on the map::
- https://www.openstreetmap.org/node/2940073921

**Comment by bege10** [comment]
> As it was assumed, your POI was deleted, so it is no longer present on the map::

Now I understand. I was misled by the fact, that obviously somebody had removed several POIs so that it looked to me as this was a general issue. Thank you very much.
